spec/models/manageiq/providers/hawkular/middleware_manager/refresher_spec.rb

```diff
@@ -3,15 +3,15 @@
     allow(MiqServer).to receive(:my_zone).and_return("default")
     auth = AuthToken.new(:name => "test", :auth_key => "valid-token", :userid => "jdoe", :password => "password")
     @ems_hawkular = FactoryGirl.create(:ems_hawkular,
-                                       :hostname        => 'localhost',
-                                       :port            => 8080,
+                                       :hostname        => 'hservices.torii.gva.redhat.com',
```

Comment:
We discussed that on IRC and it seems more reasonable to use a host where more people has access to instead of localhost. The advantage of this is that whenever a new episode is required, we need to change the VCR only for that episode, instead of re-recording everything (as seems to be the current practice for now).